src/web/app/components/question-types/question-edit-answer-form/contribution-question-edit-answer-form.component.html

```diff
@@ -6,8 +6,11 @@
               'color-neutral': responseDetails.answer === 100 || responseDetails.answer === CONTRIBUTION_POINT_NOT_SURE}"
             [ngModel]="responseDetails.answer" (ngModelChange)="triggerResponseDetailsChange('answer', $event)" [disabled]="isDisabled">
       <option [ngValue]="CONTRIBUTION_POINT_NOT_SUBMITTED"></option>
-      <option *ngFor="let point of contributionQuestionPoints" [ngValue]="point">{{ point | contributionPointDescription }}</option>
-      <option *ngIf="questionDetails.isNotSureAllowed" [ngValue]="CONTRIBUTION_POINT_NOT_SURE">Not Sure</option>
+      <option *ngFor="let point of contributionQuestionPoints" [ngValue]="point"
+              [ngClass]="{'color-positive': point > 100,
+              'color-negative': point < 100,
+              'color-neutral': point === 100}">{{ point | contributionPointDescription }}</option>
+      <option *ngIf="questionDetails.isNotSureAllowed" [ngValue]="CONTRIBUTION_POINT_NOT_SURE" [ngClass]="'color-neutral'">Not Sure</option>
```

Comment:
If there is no need to use expression, you can just say
```suggestion
      <option *ngIf="questionDetails.isNotSureAllowed" [ngValue]="CONTRIBUTION_POINT_NOT_SURE" class="color-neutral">Not Sure</option>
```

Reply:
Updated